samigo/samigo-api/src/java/xml/author/v1p2/fibTemplate.xml

```diff
@@ -38,26 +38,30 @@
         <fieldlabel>ITEM_TAGS</fieldlabel>
         <fieldentry/>
       </qtimetadatafield>
-	  <qtimetadatafield>
+      <qtimetadatafield>
         <fieldlabel>ATTACHMENT</fieldlabel>
         <fieldentry/>
       </qtimetadatafield>
+      <qtimetadatafield>
+￼        <fieldlabel>MARKERS_PAIR</fieldlabel>
```

Comment:
Please remove the wrong symbol

Reply:
Done. Sorry, they came in the patch I applied and I missed them.